docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md

```diff
@@ -171,16 +137,14 @@ future, and then those resources will join the previous category.
 
 ## Design overview
 
-In very brief: the design approach is to reduce the edge placement
-problem to kcp's TMC problem, using as many "anycast" TMC Placement
-objects (each constrained to effectively be "unicast") as are needed
-to achieve the "multicast" semantics of an EdgePlacement object.  A
-central idea of the reduction is to maintain a unique _mailbox
-workspace_ in the center for each edge cluster.
-
-However: to fully realize the goals of this PoC we will have to use a
-modified version of TMC, which differs from today's TMC in ways
-summarized below in the [Syncers](#syncers) section.
+In very brief: the design approach is to achieve the multicast
+semantics of edge placement by two layers of activity.  Between the
+two layers sit _mailbox workspaces_: these exist in the center, and
+there is one for each edge cluster.  One layer of activity runs in the
+center and relates the edge placement problems to mailbox workspace
+contents.  The other layer is _syncers_, one in each edge cluster,
+that relate the corresponding mailbox contents with their local
+cluster.
```

Comment:
Maybe this should be 'their local clusters' (plural)?

Reply:
yes, that's right, thanks.
Fixed in latest push